udata/forms/fields.py

```diff
@@ -664,7 +664,24 @@ def pre_validate(self, form):
         return True
 
 
+def field_parse(cls, value, *args, **kwargs):
+    kwargs['_form'] = Form()
+    kwargs['_name'] = 'extra'
+    field = cls(*args, **kwargs)
+    field.process_formdata([value])
+    return field.data
+
+
 class ExtrasField(Field):
+    TYPES = {
```

Comment:
Maybe `VALIDATED_PARSED_TYPES` or something? If I understand correctly, other types are supported but won't be parsed/validated.

Reply:
Exactly, they are now allowed unless register, maybe ALLOWED_TYPES as a class attribute ?

Reply:
`ALLOWED_TYPES` might be confusing, others are still allowed ;-) `SUPPORTED` is not wrong but is it clear enough?

Reply:
or `KNOWN_TYPES` ?

Reply:
👍